src/main/markdown/lifeofanissue.md

```diff
@@ -26,7 +26,7 @@ Here's the Life of an Issue, in a nutshell:
 1.  An issue is filed and has the state "New".
 2.  A GWT contributor periodically reviews and triages issues. The GWT contributor attempts to verify the issue and assigns a status based on the results.
 3.  Once an issue has been verified, it may be marked Accepted, in which case the GWT team plans to fix it, or PatchesWelcome, which means that the GWT team likely won't be able to get to it, but would be happy to review patches from the community. To submit a patch, follow [these guidelines](makinggwtbetter.html#submittingpatches).
-4.  Once an issue has been fixed, its status will be updated to Fixed orFixedNotReleased, indicating whether the fix is available in the current numbered release or, more commonly, in SVN trunk for the next release.
+4.  Once an issue has been fixed, its status will be updated to Fixed or FixedNotReleased, indicating whether the fix is available in the current numbered release or, more commonly, in SVN trunk for the next release.
```

Comment:
we could also add status labels. like status:new, status:accepted...

But indeed this should be discussed at the SC meeting (I was writing a mail about that)